panel/src/components/Collection/Item.vue

```diff
@@ -53,19 +53,24 @@
 				v-bind="button"
 			/>
 
-			<label v-if="selecting" class="k-item-options-checkbox" @click.stop>
+			<label
+				v-if="selecting !== false"
+				class="k-item-options-checkbox"
+				@click.stop
+			>
 				<input
 					ref="selector"
-					type="checkbox"
+					:checked="selected"
 					:disabled="!selectable"
+					:type="selecting === 'single' ? 'radio' : 'checkbox'"
 					@change="$emit('select', $event)"
 				/>
 			</label>
 
 			<!-- Options -->
-			<slot v-else name="options">
+			<slot name="options">
 				<k-options-dropdown
-					v-if="options"
+					v-if="options && selecting === false"
```

Comment:
`Explanation:` I think it makes sense to still allow showing a custom options slot even when selecting. E.g. when using this in the pages picker dialog, we would want to show the checkbox alongside a navigate into the page button next to it at the same time. 

Alternatively, we could keep it the old way but add another slot for UI that should still be shown when in selecting mode.